docs/git-commit-message-style-guide.md

```diff
@@ -43,8 +49,7 @@ Not all commits are complex enough to warrant a body, therefore it is optional a
 when a commit requires a bit of explanation and context. Use the body to explain the **what** and **why** of a commit,
 not the how.
 
-When writing a body, the blank line between the title and the body is required and you should limit the length
-of each line to no more than 72 characters.
+When writing a body, the blank line between the title.
```

Comment:
```suggestion
When writing a body, the blank line between the title and the body is required.
```

Reply:
Please create a ticket to make regexp meet new requirements

Reply:
Ups..